### PR TITLE
Archive unused repo

### PR DESCRIPTION
**Pull request by ryjones**
This repo has not had a code commit since 2018.

Signed-off-by: Ry Jones <ry@linux.com>

### CODE REVIEW

**Review by tmarkovski — APPROVED** [review]

**Comment by agunde406** [comment]
I agree with the move to archive this repo. We should get approvals from all Maintainers of this repo before merging:

 - [x] Andi Gunderson
 - [x] Cian Montgomery
 - [x] Dan Middleton
 - [x] Darian Plumb
 - [x] James Mitchell
 - [x] Peter Schwarz
 - [x] Richard Berg
 - [X] Ryan Beck-Buysse
 - [x] Shawn Amundson
 - [x] Tomislav Markovski

Note: Anne Chenette has asked to be removed from maintainers

**Review by rbuysse — APPROVED** [review]

**Review by rberg2 — APPROVED** [review]

**Review by dplumb94 — APPROVED** [review]

**Review by peterschwarz — APPROVED** [review]

**Review by vaporos — APPROVED** [review]

**Review by cianx — APPROVED** [review]

**Review by dcmiddle — APPROVED** [review]

**Review by jsmitchell — APPROVED** [review]

**Pull request merged**
